src/main/java/hudson/plugins/git/util/BuildChooser.java

```diff
@@ -139,14 +139,14 @@ public Build prevBuildForChangelog(String branch, @Nullable BuildData data, IGit
     }
 
     public BuildChooserDescriptor getDescriptor() {
-        return (BuildChooserDescriptor)Hudson.getInstance().getDescriptorOrDie(getClass());
+        return (BuildChooserDescriptor)Hudson.getActiveInstance().getDescriptorOrDie(getClass());
     }
 
     /**
      * All the registered build choosers.
      */
     public static DescriptorExtensionList<BuildChooser,BuildChooserDescriptor> all() {
-        return Hudson.getInstance()
+        return Hudson.getActiveInstance()
```

Comment:
Are there other implications from switching?  I was considering it, but didn't want to introduce unneeded changes to a pretty core plugin.